### ISSUE TITLE
Add grammar validation.

### ISSUE THREAD

**Issue by hajicj** [hajicj/MUSCIMarker#81]
There should be a button/keyboard shortcut that would highlight problematic CropObjects.

Some things that should be validated:

- [x] all grace-noteheads have an inlink from another notehead
- [x] all letters and numerals are a part of something (at least other_text)
- [x] all stems have an inlink from a notehead
- [x] all repeat-dots are part of a repeat
- [x] all key_signature and time_signature have outlinks

etc.

This very much depends on #50 .

**Comment by hajicj** [comment]
Being implemented through grammar cardinality mechanisms.

**Comment by hajicj** [comment]
Interface needs:

- [ ] Focusing on the problematic CropObjects (finding them by eye is a bit difficult)

**Comment by hajicj** [comment]
Closing issue, moving the remaining validation points to #88 and interface (focusing) to #86 . Will open separate ticket for improving validation UI.
